README.Rmd

```diff
@@ -16,127 +16,44 @@ options(width = 90)
 options(max.print = 200)
 ```
 
-# CRAN package search
+# Search CRAN R Packages and get their metadata
 
+<!-- badges: start -->
 [![lifecycle](https://img.shields.io/badge/lifecycle-maturing-blue.svg)](https://www.tidyverse.org/lifecycle/#maturing)
 [![Linux Build Status](https://travis-ci.org/r-hub/pkgsearch.svg?branch=master)](https://travis-ci.org/r-hub/pkgsearch)
 [![Windows Build status](https://ci.appveyor.com/api/projects/status/github/r-hub/pkgsearch?svg=true)](https://ci.appveyor.com/project/gaborcsardi/pkgsearch)
 [![CRAN status](https://www.r-pkg.org/badges/version/pkgsearch)](https://cran.r-project.org/package=pkgsearch)
 [![CRAN RStudio mirror downloads](https://cranlogs.r-pkg.org/badges/pkgsearch)](https://www.r-pkg.org/pkg/pkgsearch)
 [![Coverage status](https://codecov.io/gh/r-hub/pkgsearch/branch/master/graph/badge.svg)](https://codecov.io/github/r-hub/pkgsearch?branch=master)
+<!-- badges: end -->
 
-The pkgsearch package searches all CRAN packages. It uses a web service,
-and a careful weighting that ranks popular packages before less
-frequently used ones.
+The pkgsearch package provides two categories of services around CRAN packages:
```

Comment:
I don't think we need to separate this, we can keep the original text. Searching is more important, and we'll list the other functions in a new section.

Reply:
I'd still like to mention the other functions at the beginning of the README, because the section about searching is so long that they could be overlooked. Even if searching is more important, it'd be a pity.

Reply:
Still unhappy with the top of the README? Search is visible from there, but if you want I can tweak further of course :-)

Reply:
Yeah, I think we can just keep the search as it was before (no bold please :), and add a sentence that it also has functions to list the new / most recently updated / archived packages, and list all versions of a package.

The other functions are not terribly important, because they are also easy with base R tools.

Ideally we would have a TOC, but I guess that is hard, no? At least I had to create TOCs manually in the past...

Reply:
Do you like the new version? 

The second part of the intro is a sort of manual TOC (I don't know about automatic solutions either)

Reply:
Yes, thanks!